code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -256,11 +256,10 @@
 			dismantle_wall()
 
 /turf/closed/wall/r_wall/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
-	if(!the_rcd.canRturf)
-		return FALSE
-	return ..()
+	if(the_rcd.canRturf)
```

Comment:
switched back to variable check. It is now using a bool check of true or false instead of 0 or 1 though. Also removed the not part of the check and return false as it wasnt nessecary and didnt cause runtimes.